content/events/meeting.md

```diff
@@ -16,7 +16,11 @@ no pre-decided topics, but never run out of things to say.
 
 ## Community discussions - biweekly
 
-On every other week, or as needed, we have a community planning meeting during the coffee break time, **9:00 CET/CEST (10 EET/EEST) every Thursday**. The time
+During our biweekly community disucssions we meet and discuss what we, as Nordic-RSE can do to get RSE and RSE-minded people together to discuss the topics that interest us.
```

Comment:
Typo on "community discussions", but instead I would write "During our biweekly community meeting we discuss what we..." to avoid repetition of "discuss".

Reply:
Applied